packages/core/src/configuration/Configuration.ts

```diff
@@ -0,0 +1,47 @@
+import { CredentialProvider, Credentials } from '../credentials-provider';
+import { container, DependencyContainer, injectable } from 'tsyringe';
+
+export interface ConfigurationOptions {
+  credentials?: Credentials;
+  bus?: string;
+  api?: string;
```

Comment:
Let's use the same options the repeater uses. BTW this option must be mandatory

```suggestion
  cluster: string;
```